--- #projector(s) configuration file

#resolution of projectors, assumed to be the same for all of them
projector_resolution: [1024, 768]

#projector corner locations in world coordinates, structure [ TL, TR, BR, BL ] use y,x for each
#add new line for each projector
projector_coordinates: [[165.0,-27.0], [165.0,150.0], [0.0,126.0], [0.0,0.0]]


projector_locations: [['first', [100, 100, 500]], ['second', [-100, -100, 500]]]


# camera configuration
# camera
sense_world_points: [[0,0,0],         [0, 100, 0],   [100, 100, 0], [100, 0, 0]]

sense_camera_points:  [[1.6,8.9,315], [1,-59.6,244], [102,-59.8,243], [99,8.6,308]]
...

